requirements.txt

```diff
@@ -74,4 +74,6 @@ werkzeug==0.14.1          # via flask
 wtforms-json==0.3.3
 wtforms==2.2.1            # via flask-wtf, wtforms-json
 snowflake-sqlalchemy
-awscli
\ No newline at end of file
+awscli==1.11.180
```

Comment:
This section is not needed. Just make sure boto3 is installed

Reply:
Instead we will have to attach IAM role to get the access to KMS. I will remove this, but then the instance running the container should have access to deployment KMS.